### PR TITLE
[CoreBundle] CLI tool for emptying workspace roles.

### PR DESCRIPTION
**Pull request by ngodfraind**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Fixed tickets | #1438 

This is the CLI part of #1438

### CODE REVIEW

**Comment by LaurentGruber** [comment]
LGTM

File: main/core/Entity/Role.php [hunk 1]
```diff
@@ -280,6 +280,16 @@ public function addUser($user)
         }
     }
 
+    public function removeUser(User $user)
+    {
+        $user->getRoles()->removeElement($this);
+    }
```
Comment (Elorfin) [inline main/core/Entity/Role.php]: This seems a little weird. Why not just use/declare `$user->removeRole(Role $role)` ? Also I don't see where you use this new method.

Comment (ngodfraind) [inline main/core/Entity/Role.php]: You're right, it should be remvoed.

**Comment by Elorfin** [comment]
LGTM

**Pull request merged**